src/config/units.yaml

```diff
@@ -0,0 +1,36 @@
+cpu:
+  description: refers to CPU utilization.
+  unit: percentage
+  aggregation: avg
+memory:
+  description: refers to GB of memory utilized.
+  unit: GB
+  aggregation: avg
+disk:
+  description: refers to GB of data written/read from disk
+  unit: GB
+  aggregation: sum
+duration:
+  description: refers to the duration of the observation
+  unit: S
+  aggregation: sum
+timestamp:
+  description: refers to the time of occurrence of the observation
+  unit: ISO8601 / RFC3339 compatible timestamp
+  aggregation: None
+carbon:
+  description: an amount of carbon emitted into the atmosphere
+  unit: gCO2e
+  aggregation: sum
+energy:
+  description: amount of energy utilised by the component
+  unit: KWh
+  aggregation: sum
+embodied_emission:
+  description: Embodied Emissions of the component
+  unit: gCO2e
+  aggregation: sum
+expected_life_span:
+  description: Total Expected Lifespan of the Component in Years
+  unit: Years
```

Comment:
elsewhere we've been converting years -> seconds here (e.g. passing 126144000 to represent 4 years)

Reply:
I think it is much easier to comprehend for a human when writing a YAML file to write in just years.

Reply:
@gnanakeethan I've seen it also refered to as "expected-lifespan" in other places now so let's use "expected-lifespan" with a unit of "seconds"